doc/geany.txt

```diff
@@ -494,6 +494,27 @@ An example of a simple ``.gtkrc-2.0``::
     widget "GeanyPrefsDialog" style "geanyStyle"
 
 
+Customizing Geany's appearance using GTK+ 3 CSS
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To override GTK+ CSS styles, you can use traditional mechanisms or you
+can create a file named ``geany.css`` in the user configuration directory
+(usually ``~/.config/geany``) which will be loaded after other CSS styles
```

Comment:
Good question… it's weird, but it seems to involve `__`: http://docutils.sourceforge.net/docs/user/rst/quickref.html#internal-hyperlink-targets (see also `anonymous__` [in the *Inline markup* section](http://docutils.sourceforge.net/docs/user/rst/quickref.html#inline-markup)).